decred/decred/dcr/agenda.py

```diff
@@ -0,0 +1,220 @@
+"""
+Copyright (c) 2019-2020, Brian Stafford
```

Comment:
Hmm, I don't think we have to add Brian anywhere, because this file is all you, it's just removing that I would rather not touch. He probably just thinks it's a waste of time to discuss though, so, I probably shouldn't have said anything from the beginning.